### ISSUE TITLE
EditorOpenEvent to include keycode (int)

### ISSUE THREAD

**Issue by mihalikm93** [TatuLund/GridFastNavigation#101]
Since it is possible to add custom editor open shortcuts, it would be useful to get that information back through EditorOpenEvent.

eg.:

```
nav.addEditorOpenListener(e -> {
    if(e.getEditorOpenShortcut() == 65) { // keycode for A
    // .......
    }
});
```

**Comment by TatuLund** [comment]
Included in 2.6.0

**Comment by mihalikm93** [comment]
Feature works as expected, thanks!
